sync-android/src/main/java/com/cloudant/sync/sqlite/android/encryption/EncryptionUtils.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * IBM Confidential OCO Source Materials
+ * 
+ * 5725-I43 Copyright IBM Corp. 2006, 2013
+ * 
+ * The source code for this program is not published or otherwise
+ * divested of its trade secrets, irrespective of what has
+ * been deposited with the U.S. Copyright Office.
+ * 
+ */
+
+/* Copyright (C) Worklight Ltd. 2006-2012.  All rights reserved. */
+
+package com.cloudant.sync.sqlite.android.encryption;
+
+import android.content.Context;
+import android.graphics.Bitmap;
```

Comment:
There's a load of stuff in here which isn't crypto specific. I think we should remove it, make a reference to the class this was copied from, and document its differences.